src/mongo/db/pipeline/expression.h

```diff
@@ -506,9 +526,16 @@ class ExpressionAllElementsTrue final : public ExpressionFixedArity<ExpressionAl
 class ExpressionAnd final : public ExpressionVariadic<ExpressionAnd> {
 public:
     boost::intrusive_ptr<Expression> optimize() final;
+
```

Comment:
Please leave non-inlined methods condensed. It's more consistent with the rest of the file.